internal/detector/application_lister.go

```diff
@@ -64,8 +66,8 @@ func extractOptionArgs(cmdlineSlice []string, flags []string) ([]string, error)
 	return out, nil
 }
 
-func parseClassPath(classpath string) []string {
-	return strings.Split(classpath, ":")
+func parseClassPathWithSeparators(classpath string) []string {
+	return strings.Split(classpath, ";")
```

Comment:
Based on the actual change, I'm not sure to see the rational of changing the name of the function.

Reply:
Also, as on Linux the separator is `:` and on Windows it is `;` aren't you going to break Linux here?

Reply:
you are right: https://stackoverflow.com/a/4528456

Reply:
fixed at head